manifest.json

```diff
@@ -35,7 +35,8 @@
       ],
       "js": [
         "dist/main.js"
-      ]
+      ],
+      "css": ["css/style.css"]
```

Comment:
Glad to see this finally happen! There's some CSS in label_view (`iframe_style`, `checkbox.setAttribute('style', ...)`) that would be nice to pull into here too.

Reply:
@emcniece agree with this for sure. That will definitely happen in the future